README.md

```diff
@@ -1,2 +1,117 @@
-# physical-collectibles-nft-global-event-website
-Global Event dedicated to leading Blockchain technologies in space of producing physical collectibles.
+# Microverse Capstone Project
+## Module 1
+
+Build a Custom Website based on Template.
+
+The subject chosen: NFT Global Event Presentation. Includes custom design created for this project.
+
+
+
+## Built With
+
+- CSS HTML
+- Git version control system, Github workflows
+- javascript
+
+
+
+## Live Demo (if available)
+
+[NFT Global Event Website](https://petushka1.github.io/physical-collectibles-nft-global-event-website/)
+
+
+
+## Getting Started
+
+Clone the repository
+
+```
+git clone https://github.com/repository
+```
+Create your feature branch
+
+```
+git checkout -b <new-branch>
+```
+
+You can use default Microverse linters set or configure linting rules
+
+Expand .gitignore with more exceptions if needed
+(add test files with .extension you'd like to be ignored by Git)
+
+Customize template design.
+Push your repository to GitHub.
+(all graphics are open-source and free to use).
+
+```
+git add .
+git commit -m "Commit description"
+git push origin <your-branch>
+```
+
+Use Participants Section to customize your own dynamic list of cards.
+- card is automatically generated and appended to document. To generate a new card you need to add a new Object to Array of Cards.
+
+*Example:*
+
+```
+{
+  image: 'img/photo5.png',
+  alt: 'Saachi',
+  name: 'Kevin Roberts',
+  position: 'Former executive chairman of Saatchi & Saatchi and speaker',
+  info: 'Mauris ut mauris fringilla, placerat dolor id, condimentum urna. Duis interdum arcu ut consectetur porttitor.',
+}
+```
+- You will not be required for any extra manipulations to set up cards popups. Popup is generated and receives all data from the ```cardsArr``` item.
+
+Open pull request
+
+Check Linters:
+- Lighthouse
+- Webhint
+- Stylelint
+
+Debug HTML Javascript and CSS files if checks were unsuccessful
+
+
+
+
+## Author
+
+👤 **Nadia Levina**
+
+- GitHub: [@petushka1](https://github.com/petushka1)
+
+
+
+## Inspiration
+
+👤 **Cindy Shin**
```

Comment:
Appropriate credit to Cindy Shin in your Acknowledgments section, as required by the Creative Commons License :heavy_check_mark: